python/test/t_GaussianLinearCalibration_std.py

```diff
@@ -2,47 +2,110 @@
 
 from __future__ import print_function
 import openturns as ot
-import openturns.testing as ott
-import math as m
-import sys
+from openturns.testing import assert_almost_equal
 
 ot.TESTPREAMBLE()
 ot.PlatformInfo.SetNumericalPrecision(5)
 
-m = 10
-x = [[0.5 + i] for i in range(m)]
-
-inVars = ["a", "b", "c", "x"]
-formulas = ["a + b * exp(c * x)", "(a * x^2 + b) / (c + x^2)"]
-model = ot.SymbolicFunction(inVars, formulas)
-p_ref = [2.8, 1.2, 0.5]
-params = [0, 1, 2]
-modelX = ot.ParametricFunction(model, params, p_ref)
-y = modelX(x)
-y += ot.Normal([0.0]*2, [0.05]*2, ot.IdentityMatrix(2)).getSample(m)
-candidate = [1.0]*3
-priorCovariance = ot.CovarianceMatrix(3)
-for i in range(3):
-    priorCovariance[i, i] = 3.0 + (1.0 + i) * (1.0 + i)
-    for j in range(i):
-        priorCovariance[i, j] = 1.0 / (1.0 + i + j)
-errorCovariance = ot.CovarianceMatrix(2)
-for i in range(2):
-    errorCovariance[i, i] = 2.0 + (1.0 + i) * (1.0 + i)
-    for j in range(i):
-        errorCovariance[i, j] = 1.0 / (1.0 + i + j)
-globalErrorCovariance = ot.CovarianceMatrix(2 * m)
-for i in range(2 * m):
-    globalErrorCovariance[i, i] = 2.0 + (1.0 + i) * (1.0 + i)
-    for j in range(i):
-        globalErrorCovariance[i, j] = 1.0 / (1.0 + i + j)
-
-methods = ["SVD", "QR", "Cholesky"]
-for method in methods:
-    print("method=", method)
-    algo = ot.GaussianLinearCalibration(modelX, x, y, candidate, priorCovariance, errorCovariance, method)
-    algo.run()
-    print("result=", algo.getResult())
-    algo = ot.GaussianLinearCalibration(modelX, x, y, candidate, priorCovariance, globalErrorCovariance, method)
-    algo.run()
-    print("result (global)=", algo.getResult())
+def matrixToSample(matrix):
+    '''Converts a matrix into a Sample, so that we can  
+    assert_almost_equal a Matrix'''
+    size = matrix.getNbRows()
+    dimension = matrix.getNbColumns()
+    sample = ot.Sample(size,dimension)
+    for i in range(size):
+        for j in range(dimension):
+            sample[i,j] = matrix[i,j]
+    return sample
+
+def testCalibrationtExponential():
+    '''
```

Comment:
Same remark here : no need to define a function

Reply:
OK.